source/sggsym.h

```diff
@@ -0,0 +1,108 @@
+#ifndef SGGSYM_H
+#define SGGSYM_H
+
+#include "Symbiont.h"
+#include "SymWorld.h"
+
+class SggSymbiont: public Symbiont {
+protected:
+  double sgg_donate = 0;
+
+
+public:
+  SggSymbiont(emp::Ptr<emp::Random> _random, emp::Ptr<SymWorld> _world, emp::Ptr<SymConfigBase> _config, double _intval=0.0, double _donation = 0.0, double _points = 0.0 ) : Symbiont(_random, _world, _config, _intval, _points),sgg_donate(_donation)
+  {}
+  SggSymbiont(const SggSymbiont &) = default;
+  SggSymbiont(SggSymbiont &&) = default;
+  SggSymbiont() = default;
+
+
+  SggSymbiont & operator=(const SggSymbiont &) = default;
+  SggSymbiont & operator=(SggSymbiont &&) = default;
+
+
+  double Getdonation() {return sgg_donate;}
```

Comment:
Could you rename these functions to GetDonation and SetDonation for consistency with the style of the rest of the code?